package/MDAnalysis/lib/c_gridsearch.pyx

```diff
@@ -0,0 +1,647 @@
+#cython: cdivision=True

```

Comment:
We need to remember "add due.cite" for PRs in the future. For this PR I'd say that it can be done once @ayushsuhane works on the docs, so I wouldn't worry about it immediately. 

The comment header is important to me, though, because we want to make sure that we don't appear to take code that we cannot legally use or to not give proper attribution. Adding comments from the first commit shows that we are taking attribution seriously.